specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2022-04-01-preview/Settings.json

```diff
@@ -333,14 +333,28 @@
     "EntityAnalyticsProperties": {
       "description": "EntityAnalytics property bag.",
       "properties": {
-        "isEnabled": {
```

Comment:
removal of properties is seen as a breaking change and should get flagged as a breaking change when this goes to main if this property existed in the previous API version.  You'll have to go through the breaking change process outlined in the PR description and talk to those folks if you need to remove the boolean.